### PR TITLE
Explicitely serialize as text/turtle

### PR DESCRIPTION
**Pull request by angelaraya**
When a named graph is given, N3.Writer will use TriG notation
inseted of turtle. From my testing this breaks authentication/authorization
when using other libraries.

### CODE REVIEW

**Comment by angelaraya** [comment]
Upon further testing, the solution is not as straight forward as I thought it to be. Moving to draft for now.

**Comment by elf-pavlik** [comment]
I think if an instance of [Default Graph](https://rdf.js.org/data-model-spec/#defaultgraph-interface) is used for a graph name it should work fine. Can you provide a snippet illustrating when you encounter the problem?

**Comment by angelaraya** [comment]
@elf-pavlik For some reason when pulling `profile/card` from a pod, modifying it and then serializing it results in a trig serialization. I'd assume that the parser is the one that's adding a named graph but I'm not certain. I added the change as an optional flag set to false as to not disrupt anything else.

File: packages/utils/src/fetch.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ async function unwrappedRdfFetch(
     requestInit.headers = { Accept: 'text/turtle', ...requestInit.headers };
   }
   const response = await whatwgFetch(iri, requestInit);
-  const rdfResponse = { ...response } as RdfResponse;
+  const rdfResponse = response.clone() as RdfResponse;
```
Comment (angelaraya) [inline packages/utils/src/fetch.ts:35]: Copying a response with object destructuring does not work, the `.clone` method has to be used. However this breaks the tests as the mocks do not implement it, @elf-pavlik can you take a look at fixing the mocks?

File: packages/utils/src/link-header.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import LinkHeader from 'http-link-header';
 
 export function getApplicationRegistrationIri(linkHeaderText: string): string {
   const links = LinkHeader.parse(linkHeaderText).refs;
-  return links.find((link) => link.rel === 'http://www.w3.org/ns/solid/interop#registeredApplication')?.anchor;
+  return links.find((link) => link.rel === 'http://www.w3.org/ns/solid/interop#registeredAgent')?.anchor;
```
Comment (angelaraya) [inline packages/utils/src/link-header.ts:5]: @elf-pavlik can you double check whether this should be `Agent` or `Application` or both? In the service side of things it is checking for `Agent` https://github.com/janeirodigital/sai-impl-service/blob/3b0def205c234f355daee36f619ae2fbc74a7304/packages/service/src/handlers/agents-handler.ts#L42

Comment (elf-pavlik) [reply packages/utils/src/link-header.ts:5]: it should be Agent

**Comment by elf-pavlik** [comment]
I'm going to deal with ShEx dependencies in dedicated PR.

**Pull request merged**